README.md

```diff
@@ -3,230 +3,161 @@
 
 ## Prerequisites
 
-The following assumes that you have completed the steps for setting up your local environment as well as creating an account and project.  If you have not done this you must follow the instructions here:
+* [Your Lightbend Cloudstate Account](https://docs.lbcs.io/gettingstarted/account.html)
+* Install [csctl](https://docs.lbcs.io/getting-started/set-up-development-env.html)
 
-* [Setting Up your Machine](https://docs.lbcs.dev/gettingstarted/setup.html)
-   * as well as the [Developer prerequisites](https://docs.lbcs.dev/developing/developing.html#prerequisites)
-   * Install [nvm](https://github.com/nvm-sh/nvm#install--update-script) (node version manager)
-      * We recommend v0.34.0 or later.  (Check with `nvm --version`)
-   * Install [npm](https://www.npmjs.com/get-npm) (node package manager)
-      * We recommend v6.14.3 or later.  (Check with `npm -v`)
-   * You also need to install the protobuf compiler.
-      * We recommend using v3.0.0 or later.  (Check with `protoc --version`)
-      * Mac OS X `brew install protobuf`
-      * Linux `sudo apt install protobuf-compiler`
-      * Or [alternatively](https://developers.google.com/protocol-buffers/docs/downloads) (src and bins)
-* [Your Lightbend Cloudstate Account](https://docs.lbcs.dev/gettingstarted/account.html)
-* [Creating a Project](https://docs.lbcs.dev/gettingstarted/project.html)
+## Deploying
 
-## Sample application layout
+The following steps uses `csctl` to deploy both the [shopping-cart frontend
```

Comment:
A few typos corrected here, corrections in italics:

The following steps *use* `csctl` to deploy both the [shopping-cart frontend
service](https://github.com/cloudstateio/samples-ui-shoppingcart) *and* the
CloudState shopping-cart service defined in this repo. We'll be using the
pre-built Docker images for the frontend and the shopping-cart, but feel free to
build and use your own Docker images where appropriate.